NullPointerException on twitter login for react-native-image-picker integrated app

Hi @corymsmith Im getting a null pointer exception, if i use [react-native-image-picker](https://github.com/marcshilling/react-native-image-picker) in my [Fabric twitter login] (https://github.com/corymsmith/react-native-fabric) integrated app. 
The twitter login is working fine but, When i try to pick any image form gallery / camera roll it shows an alert of unfortunately app has stopped and get closed.
When i check the stack trace its showing null pointer expection at

> Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.twitter.sdk.android.core.identity.TwitterLoginButton.onActivityResult(int, int, android.content.Intent)' on a null object reference

![c5bde1a2-1780-11e6-88eb-4a684ad2a910](https://cloud.githubusercontent.com/assets/11032434/15205293/9969209a-1833-11e6-8ce9-895a3e9158e5.png)

Thanks in Advance

--- a/android/src/main/java/com/smixx/fabric/SMXTwitter.java
+++ b/android/src/main/java/com/smixx/fabric/SMXTwitter.java
@@ -81,7 +81,9 @@ public class SMXTwitter extends ReactContextBaseJavaModule {
     }
 
     public void onActivityResult(int requestCode, int resultCode, Intent data) {
-        loginButton.onActivityResult(requestCode, resultCode, data);
+        if (loginButton != null) {
+            loginButton.onActivityResult(requestCode, resultCode, data);
+        }
     }
 
     private boolean hasValidKey(String key, ReadableMap options) {

